pkg/features/features.go

```diff
@@ -90,6 +90,14 @@ const (
 	// beta: v1.89.0
 	// GA: v1.90.0
 	UseGardenerNodeAgent featuregate.Feature = "UseGardenerNodeAgent"
+
+	// UseNamespacedCloudProfile enables the usage of the NamespacedCloudProfile API object
+	// nodes.
+	// owner: @timuthy @benedictweis
+	// alpha: v1.91.0
+	// beta: tbd
+	// GA: tbd
```

Comment:
You could drop these lines. We don't specify them usually

Reply:
Thanks for the info. Removed in https://github.com/gardener/gardener/pull/9440/commits/bc2bd226df5058cdde6c446ccabd89b024c97d49.